overlap: 4
trajectory_length: 30

robot_dimension: 9
#robot_dimension: 7

joint_order: ['base_joint_1', 'base_joint_2', 'arm_joint_1', 'arm_joint_2', 'arm_joint_3', 'arm_joint_4', 'arm_joint_5', 'arm_joint_6', 'arm_joint_7']
#joint_order: ['arm_joint_1', 'arm_joint_2', 'arm_joint_3', 'arm_joint_4', 'arm_joint_5', 'arm_joint_6', 'arm_joint_7']

fixed_joints: ['table_joint_1', 'table_joint_2']
#fixed_joints: ['base_joint_1', 'base_joint_2', 'table_joint_1', 'table_joint_2']

fixed_values: [0, 0]
#fixed_values: [-0.5, 0.1, 0, 0]
